apps/molgenis-components/src/client/client.ts

```diff
@@ -1,9 +1,11 @@
 import axios, { AxiosError, AxiosResponse } from "axios";
```

Comment:
client is next candidate to be moved to meta-data-utils, right?

Reply:
hmm i would like to  separate the sync (side effect free ) util functions from the async ( fetch , caching ) functions , but could be separate file instead of separate module , the 'new' ssr functions are more strict about this separation. i thinks this is important else you end up with every thing becoming async. The ssr first does a fetch for all the meta data and then uses sync util function.